controllers/suite_test.go

```diff
@@ -90,12 +92,18 @@ var _ = BeforeSuite(func() {
 	err = enterpriseApi.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = enterpriseApi.AddToScheme(scheme.Scheme)
+	err = enterpriseApiV3.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
 	err = enterpriseApi.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+	//err = enterpriseApiV3.AddToScheme(scheme.Scheme)
+	//Expect(err).NotTo(HaveOccurred())
+	//
+	//err = enterpriseApiV3.AddToScheme(scheme.Scheme)
+	//Expect(err).NotTo(HaveOccurred())
+
```

Comment:
probably needs to be removed?

Reply:
old code , yes need some changes we can take this in separate CR

Reply:
done